deploy/aks/monitoring/perf/prometheus-pushgateway-configmap.yaml

```diff
@@ -0,0 +1,15 @@
+kind: ConfigMap
+apiVersion: v1
+data:
+  prometheus-config: |-
+    global:
+      scrape_interval: 5s
```

Comment:
Is 5 seconds the default value? Seems too frequent, no?

Reply:
Yeah, making it 5 minute for perf test.